casepro/settings_production.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import unicode_literals
+import os
+
+# import our default settings
+from settings_common import *  # noqa
+
+SECRET_KEY = os.environ.get('SECRET_KEY', 'REPLACEME')
+if os.environ.get('DEBUG', 'False') == 'True':  # envvars are strings
```

Comment:
the second param is the default if the envvar isn't present. Make more sense now?